1.	Identify-Basic-Internet-Problems-with-ping.md

```diff
@@ -0,0 +1,113 @@
+![Sonar screen](https://media.licdn.com/mpr/mpr/jc/AAEAAQAAAAAAAAfoAAAAJDRmY2I4ZmE1LTk4ZmQtNGUzOS1hOWQ4LTNkMGJkOWQxNjNkYg.png width="400")
```

Comment:
The image link is broken. I am getting this:

![untitled4](https://cloud.githubusercontent.com/assets/5607371/14708670/ee92d9da-07e9-11e6-8148-a4af2c1602ab.png)